templates/nro/.circleci/config.yml.tmpl

```diff
@@ -1,3 +1,4 @@
+{{- if not .Env.IS_CONFIG_UPDATE }}
 ---
 version: 2
```

Comment:
Do we need these lines now that there is a header template?

Reply:
Yeah ideally we put the header template in one place. However I didn't want to make changes to [this script](https://github.com/greenpeace/planet4-nro-generator/blob/master/bin/init_github_repo.sh#L149), as the NRO generation is kinda complex and it seemed it would be too much to add to this PR. Currently that uses just `dockerize` with one template.

Maybe we can consider not using `dockerize` for this? As what dockerize does is use the environment to populate configuration files, but in this case we simply want to pass values to a template. Instead we could base the generator script on a config file for each generated NRO, where it would just read that config without having to put it in the environment. Then we can also check in that config file and use it for updating `config.yml` as well.